recipes/large-image-converter/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "large-image-converter" %}
+{% set version = "1.8.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3a132f2c722f54dbb772e3e7a020bbe38a6876642071544e3a677807f42b8f4a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  # [win, py < 36]
```

Comment:
I'm not sure the `,` works there. Are you skipping this on Windows b/c of a missing dependency? If so, it is OK to make it noarch, add a `python >=3.6` in host and run, and let Windows fail b/c of the missing dep. Users trying to install it on Windows will get an error message about it.

The advantage of noarch is that we build this only once. Note that if you have compiled extensions or scripts in your setup.py we cannot use noarch. (But that does not seem to be the case here.)

Reply:
> Are you skipping this on Windows b/c of a missing dependency?

Yes, libvips is a dependency and it is not built on Windows per https://github.com/conda-forge/libvips-feedstock/issues/29

> If so, it is OK to make it noarch, add a python >=3.6 in host and run, and let Windows fail b/c of the missing dep

That's what I did in  4ad0c373786a2976bf1d3c692cb149a2020ada7c but I got this listing error: https://github.com/conda-forge/staged-recipes/pull/17167#issuecomment-987128797 and

Reply:
or what do you mean by:

> let Windows fail b/c of the missing dep. 

I have to "skip" windows, I can't just let it "fail"

Reply:
Or conda-forge isn't even building on 3.6 these day so I can just free the Python constraint, can't I?

> 2021-10-04: python 3.6 is now dropped when building conda-forge packages

From https://conda-forge.org/docs/user/announcements.html

Reply:
Hopefully this is resolved in 577c998

Reply:
A noarch pkg will only be built on linux and thus the failure on windows here will be irrelevant. @ocefpaf is saying you should convert this to noarch; for that to happen you need to add `noarch: python`, but you *must* specify `python > 3.6` in run and host and can *NOT* have any os restrictions (e.g. skips: true # [win]) anywhere. They will approve the pkg regardless if the windows build fails or passes.

Reply:
You still need to designate it as noarch.

Reply:
(they will approve it even if the windows build is failing, so just designate it as noarch, remove the win os restrictions)

Reply:
Okay I just pushed this change in 570d9d7

Reply:
@ngam, I'm still kind of confused why we did it this way. I was mostly following what pyvips (the dependency with missing support on Windows) was doing here: https://github.com/conda-forge/pyvips-feedstock/blob/1125285216bc3d8867b794c155de1e2cd0e88c8b/recipe/meta.yaml#L27

AFAIK, `pyvips` is also pure python / noarch (with libvips being the real dependency trouble maker)

Reply:
@hmaarrfk, do you have any feedback for this?

Reply:
@banesullivan I am actually new to this (and I was just commenting on my understanding of how they prefer to do things here). The rationale, I think, is about conserving resources and simplicity. The package itself is noarch and should be noarch; the problem is with its dependencies. So the package remains noarch, and users on Windows will be informed that they cannot have it because of a dependency. This way, actually, it could incentivize users to submit an issue or push for changes at the dependency causing problems. This is just my interpretation. Both @hmaarrfk and @ocefpaf are superb and helpful, and know significantly more than I do, so you're in safe hands here :)

Reply:
I'm even newer to this 😉  

Ahh, that rationale makes sense to me! I just like to see those little green checks instead of the red X's so I was confused why we'd want failing builds... 

Either way, I'm eager to move forward with these recipes and if the one recipe with the failing windows dependency is holding this back, I can open a separate PR for that recipe

Reply:
Hopefully one of the ones involved here (ocefpaf, hmaarrfk, or beckermr) can merge this fast for you. Usually, I would say, ping them after 1 week using the "at staged-recipes" you used below.

Isn't windows such a pain 😞 

This looks like a valuable collection of packages btw, thanks for the good work! I will have a look to see if I can make use of them!